sql/src/test/java/org/apache/druid/sql/avatica/DruidAvaticaHandlerTest.java

```diff
@@ -101,7 +101,7 @@
 import java.util.concurrent.Executors;
 import java.util.concurrent.ThreadLocalRandom;
 
-public class DruidAvaticaHandlerTest extends CalciteTestBase
+public abstract class DruidAvaticaHandlerTest extends CalciteTestBase
```

Comment:
nit: I wonder if this test would be better using `@RunWith(JUnitParamsRunner.class)`  with the different handler implementations instead of making it an abstract class

Reply:
Would that not make the test harder to read? You mean parameterising each test case? I was going for a lighter touch approach, the abstracted methods are called in setUp, which would then need rewriting